user_docs/en/userGuide.t2t

```diff
@@ -1271,10 +1271,10 @@ The dialog also contains Add, Edit and Remove buttons.
 
 To add a new rule to the dictionary, press the Add button, and fill in the fields in the dialog box that appears and then press Ok.
 You will then see your new rule in the list of rules.
-However to make sure your rule is actually saved, make sure to press Ok to exit the dictionary dialog all together once you have finished adding/editing rules.
+However, to make sure your rule is actually saved, make sure to press Ok to exit the dictionary dialog altogether once you have finished adding/editing rules.

```

Comment:
I wonder if 'altogether' is needed at all. Or perhaps it could be replaced with 'completely'.